test/integration-test/src/tests/load.rs

```diff
@@ -13,6 +15,30 @@ use aya::{
 const MAX_RETRIES: usize = 100;
 const RETRY_DURATION: time::Duration = time::Duration::from_millis(10);
 
+#[test]
+fn tc_name_limit() {
+    let clsact_exists = clsact_qdisc_exists("lo").unwrap();
+    if !clsact_exists {
+        qdisc_add_clsact("lo").unwrap();
+    }
+
+    let mut bpf = Bpf::load(crate::TC_NAME_LIMIT_TEST).unwrap();
+
+    // This magic number (256) is derived from the fact that the kernel
```

Comment:
this comment is somewhat misplaced; this 256 is just defined by the fact that this is the name used in the probe code.